app/assets/stylesheets/application.scss

```diff
@@ -15,3 +15,25 @@
  *= require foundation_and_overrides
 
  */
+@import url(https://fonts.googleapis.com/css?family=Alfa+Slab+One);
+
+.home-page {
+  width: 100%;
+  height: 100%;
+  background: url(/assets/panama_lineup.jpg);
+  background-size: cover;
+}
+
+.welcome {
+  border: 1px solid white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Properties should be ordered background-color, border, margin-top, padding